src/globus_sdk/services/flows/client.py

```diff
@@ -270,6 +276,143 @@ def list_flows(
+            a more detailed explanation of the flow definition, see
+            `Authoring Flows <https://globus-automate-client.readthedocs.io/en/latest/authoring_flows.html>`_
+        :type definition: dict, optional
+        :param input_schema: A JSON Schema to which Flow Invocation input must conform
+        :type input_schema: dict, optional
+        :param subtitle: A concise summary of the flow’s purpose.
+        :type subtitle: str (0 - 128 characters), optional
+        :param description: A detailed description of the flow's purpose for end user
+            display.
+        :type description: str (0 - 4096 characters), optional
+        :param flow_owner: An Auth Identity URN to set as flow owner; this must match
+            the Identity URN of the entity calling `update_flow`
+        :type flow_owner: str, optional
+        :param flow_viewers: A set of Principal URN values, or the value "public"
+            indicating entities who can view the flow
+
+            .. dropdown:: Example Values
+
+                .. code-block:: json
+
+                    [ "public" ]
+
+                .. code-block:: json
+
+                    [
+                        "urn:globus:auth:identity:b44bddda-d274-11e5-978a-9f15789a8150",
+                        "urn:globus:groups:id:c1dcd951-3f35-4ea3-9f28-a7cdeaf8b68f"
+                    ]
+
+        :type flow_viewers: list[str], optional
+        :param flow_starters: A set of Principal URN values, or the value
+            "all_authenticated_users" indicating entities who can initiate a *Run* of
+            the flow
+
+            .. dropdown:: Example Values
+
+                .. code-block:: json
+
+                    [ "all_authenticated_users" ]
+
+
+                .. code-block:: json
+
+                    [
+                        "urn:globus:auth:identity:b44bddda-d274-11e5-978a-9f15789a8150",
+                        "urn:globus:groups:id:c1dcd951-3f35-4ea3-9f28-a7cdeaf8b68f"
+                    ]
+
+        :type flow_starters: list[str], optional
+        :param flow_administrators: A set of Principal URN values indicating entities
+            who can perform administrative operations on the flow (create, delete,
+            update)
+
+            .. dropdown:: Example Value
+
+                .. code-block:: json
+
+                    [
+                        "urn:globus:auth:identity:b44bddda-d274-11e5-978a-9f15789a8150",
+                        "urn:globus:groups:id:c1dcd951-3f35-4ea3-9f28-a7cdeaf8b68f"
+                    ]
+
+        :type flow_administrators: list[str], optional
+        :param keywords: A set of terms used to categorize the flow used in query and
+            discovery operations
+        :type keywords: list[str] (0 - 1024 items), optional
+        :param additional_fields: Additional Key/Value pairs sent to the create API
+        :type additional_fields: dict or str -> any, optional
```

Comment:
This wording confuses me. Should this be "dict _of_ str -> any"?

```suggestion
        :type additional_fields: dict of str -> any, optional
```